<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="TaskView" name="task_treeView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="TomatoWidget" name="tomato_widget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="new_action"/>
    <addaction name="open_action"/>
    <addaction name="save_action"/>
    <addaction name="saveAs_action"/>
    <addaction name="close_action"/>
    <addaction name="separator"/>
    <addaction name="properties_action"/>
    <addaction name="separator"/>
    <addaction name="quit_action"/>
   </widget>
   <widget class="QMenu" name="task_menu">
    <property name="title">
     <string>T&amp;ask</string>
    </property>
    <addaction name="activateTask_action"/>
    <addaction name="separator"/>
    <addaction name="addTask_action"/>
    <addaction name="editTask_action"/>
    <addaction name="removeTask_action"/>
    <addaction name="removeAllTasks_action"/>
    <addaction name="separator"/>
    <addaction name="expandAllTasks_action"/>
    <addaction name="collapseAllTasks_action"/>
   </widget>
   <widget class="QMenu" name="tools_menu">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="settings_action"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="about_action"/>
    <addaction name="aboutQt_action"/>
   </widget>
   <widget class="QMenu" name="tomato_menu">
    <property name="title">
     <string>T&amp;omato</string>
    </property>
    <addaction name="startTomato_action"/>
    <addaction name="stopTomato_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="tomato_menu"/>
   <addaction name="task_menu"/>
   <addaction name="tools_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="about_action">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="aboutQt_action">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="settings_action">
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
  </action>
  <action name="activateTask_action">
   <property name="text">
    <string>Ac&amp;tivate</string>
   </property>
  </action>
  <action name="addTask_action">
   <property name="text">
    <string>&amp;Add...</string>
   </property>
  </action>
  <action name="removeTask_action">
   <property name="text">
    <string>&amp;Remove...</string>
   </property>
  </action>
  <action name="removeAllTasks_action">
   <property name="text">
    <string>Remove a&amp;ll...</string>
   </property>
   <property name="toolTip">
    <string>Remove all</string>
   </property>
  </action>
  <action name="editTask_action">
   <property name="text">
    <string>&amp;Edit...</string>
   </property>
  </action>
  <action name="expandAllTasks_action">
   <property name="text">
    <string>E&amp;xpand all</string>
   </property>
  </action>
  <action name="collapseAllTasks_action">
   <property name="text">
    <string>&amp;Collapse all</string>
   </property>
  </action>
  <action name="startTomato_action">
   <property name="text">
    <string>S&amp;tart tomato</string>
   </property>
   <property name="toolTip">
    <string>Start tomato</string>
   </property>
  </action>
  <action name="stopTomato_action">
   <property name="text">
    <string>Sto&amp;p tomato</string>
   </property>
   <property name="toolTip">
    <string>Stop tomato</string>
   </property>
  </action>
  <action name="quit_action">
   <property name="text">
    <string>&amp;Quit...</string>
   </property>
  </action>
  <action name="open_action">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="saveAs_action">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
   <property name="toolTip">
    <string>Save as</string>
   </property>
  </action>
  <action name="new_action">
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
  </action>
  <action name="properties_action">
   <property name="text">
    <string>&amp;Properties...</string>
   </property>
   <property name="toolTip">
    <string>Properties</string>
   </property>
  </action>
  <action name="close_action">
   <property name="text">
    <string>&amp;Close...</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaskView</class>
   <extends>QTreeView</extends>
   <header location="global">taskview.h</header>
  </customwidget>
  <customwidget>
   <class>TomatoWidget</class>
   <extends>QWidget</extends>
   <header>tomatowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>task_treeView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
